examples/filtered-download.py

```diff
@@ -0,0 +1,60 @@
+from dash import Dash, dcc, html, Input, Output
+import plotly.express as px
+import pandas as pd
+import dash_bootstrap_components as dbc
+from base64 import b64encode
+import io
+
+app = Dash(__name__, external_stylesheets=[dbc.themes.BOOTSTRAP])
+
+df = pd.read_csv(
+    "https://raw.githubusercontent.com/plotly/datasets/master/us-cities-top-1k.csv"
+)
+df = df[["City", "State", "Population"]]
+
+dropdown_state = dcc.Dropdown(
+    id="filtered-download-x-dropdown", options=df.State, value="New York"
+)
+
+download_button = html.A(
+    html.Button("Download as HTML"),
```

Comment:
Small suggestion - if this is a `dbc.Button` instead, it will automatically get the Bootstrap style and theme color

`dbc.Button("Download as HTML", className="mt-2"),`